Django_Blog/blog/urls.py

```diff
@@ -1,5 +1,10 @@
 from django.urls import path
-from .views import index
+from .views import index,AuthorListView,AuthorDetailView,BlogListView,BlogDetailView,CommentFormView
 urlpatterns = [
-    path('',index.as_view(),name='index')
+    path('',index.as_view(),name='index'),
+    path('blogger/',AuthorListView.as_view(),name='authorlist'),
+    path('blogger/<int:pk>/',AuthorDetailView.as_view(),name='authordetail'),
+    path('blog/',BlogListView.as_view(),name='bloglist'),
```

Comment:
Please refer the document properly. There is properly mentioned that for `BlogListView` the url must be `blog/blogs`